src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,27 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  filterItem: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, filterItem }) => {
+  const filterItemLowerCase = filterItem.toLowerCase().trim();
+  const visibleMovies = movies
+    .filter(({ title, description }) => {
+      const titleLowerCase = title.toLowerCase();
+      const descriptionlowerCase = description.toLowerCase();
+
+      const inTitle = titleLowerCase.includes(filterItemLowerCase);
+      const inDescription = descriptionlowerCase.includes(filterItemLowerCase);
+
+      return inTitle || inDescription;
+    });
```

Comment:
1. I believe it will be better to have this logic in `App`, Movie list must be prepared before component will render & you don't need extra props
2. Pay attention on variables naming
```suggestion
  const visibleMovies = movies
    .filter(({ title, description }) => {
      const preparedTitle = title.toLowerCase();
      const descriptionlowerCase = description.toLowerCase();

      const isIncludeTitle = titleLowerCase.includes(filterItemLowerCase);
      const inDescription = descriptionlowerCase.includes(filterItemLowerCase);

      return inTitle || inDescription;
    });
```